src/autocollapse/autocollapse.js

```diff
@@ -10,9 +10,11 @@ export default {
 };
 
 let ig;
+let df;
```

Comment:
I would name this `autocollapseDeletedFiles`, to make it consistent with the config value :)